pyquidax/quidax.py

```diff
@@ -62,3 +62,36 @@ def withdrawal_fee(self, currency: Currency):
 
 class AsyncQuidax:
```

Comment:
Blessing it's not just about coping and pasting. I want you to understand why we do what we do. This issue is about implementing AsyncQuidax you've already seen an empty class defined so why create a new one with the same name